--- mc.ext.ini	2024-12-30 03:54:42.881221823 +0000
+++ mc.ext.ini.new	2024-12-30 03:54:38.652873036 +0000
@@ -58,7 +58,7 @@
 #            and \( \) stand for literal ( ).
 #
 #            Example:
-#                Regex=\.t(ar\.lzma|lz)$
+#                Regex=\\.t(ar\\.lzma|lz)$
 #            matches *.tar.lzma or *.tlz.
 #
 #        RegexIgnoreCase [true|false]
@@ -188,7 +188,7 @@
 
 ### GIT Repo ###
 [gitfs changeset]
-Regex=^\[git\]
+Regex=^\\[git\\]
 Open=%cd %p/changesetfs://
 View=%cd %p/patchsetfs://
 
@@ -200,7 +200,7 @@
 
 # .tgz, .tpz, .tar.gz, .tar.z, .tar.Z
 [tar.gzip]
-Regex=\.t([gp]?z|ar\.g?[zZ])$
+Regex=\\.t([gp]?z|ar\\.g?[zZ])$
 Include=tar.gz
 
 [ipk]
@@ -217,13 +217,13 @@
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.bzip
 
 [tar.bzip2]
-Regex=\.t(ar\.bz2|bz2?|b2)$
+Regex=\\.t(ar\\.bz2|bz2?|b2)$
 Open=%cd %p/utar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.bzip2
 
 # .tar.lzma, .tlz
 [tar.lzma]
-Regex=\.t(ar\.lzma|lz)$
+Regex=\\.t(ar\\.lzma|lz)$
 Open=%cd %p/utar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.lzma
 
@@ -234,19 +234,19 @@
 
 # .tar.lz4, .tlz4
 [tar.lz4]
-Regex=\.t(ar\.lz4|lz4)$
+Regex=\\.t(ar\\.lz4|lz4)$
 Open=%cd %p/utar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.lz4
 
 # .tar.xz, .txz
 [tar.xz]
-Regex=\.t(ar\.xz|xz)$
+Regex=\\.t(ar\\.xz|xz)$
 Open=%cd %p/utar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.xz
 
 # .tar.zst, .tzst
 [tar.zst]
-Regex=\.t(ar\.zst|zst)$
+Regex=\\.t(ar\\.zst|zst)$
 Open=%cd %p/utar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.zst
 
@@ -258,7 +258,7 @@
 
 # .qpr/.qpk - QNX Neutrino package installer files
 [tar.qpr]
-Regex=\.qp[rk]$
+Regex=\\.qp[rk]$
 Open=%cd %p/utar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar.qpr
 
@@ -269,7 +269,7 @@
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view tar
 
 [arj]
-Regex=\.a(rj|[0-9][0-9])$
+Regex=\\.a(rj|[0-9][0-9])$
 RegexIgnoreCase=true
 Open=%cd %p/uarj://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view arj
@@ -287,7 +287,7 @@
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view ha
 
 [rar]
-Regex=\.r(ar|[0-9][0-9])$
+Regex=\\.r(ar|[0-9][0-9])$
 RegexIgnoreCase=true
 Open=%cd %p/urar://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view rar
@@ -335,7 +335,7 @@
 Include=cpio
 
 [initrd]
-Regex=^(initramfs.*\.img|initrd(-.+)?\.img(-.+)?)$
+Regex=^(initramfs.*\\.img|initrd(-.+)?\\.img(-.+)?)$
 Include=cpio
 
 [7zip]
@@ -345,32 +345,32 @@
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view 7z
 
 [patch]
-Regex=\.(diff|patch)$
+Regex=\\.(diff|patch)$
 Open=%cd %p/patchfs://
 View=%view{ascii} /usr/lib/mc/ext.d/misc.sh view cat
 
 [patch.gz]
-Regex=\.(diff|patch)\.(gz|Z)$
+Regex=\\.(diff|patch)\\.(gz|Z)$
 Open=%cd %p/patchfs://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view gz
 
 [patch.bz2]
-Regex=\.(diff|patch)\.bz2$
+Regex=\\.(diff|patch)\\.bz2$
 Open=%cd %p/patchfs://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view bz2
 
 [patch.xz]
-Regex=\.(diff|patch)\.xz$
+Regex=\\.(diff|patch)\\.xz$
 Open=%cd %p/patchfs://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view xz
 
 [patch.zst]
-Regex=\.(diff|patch)\.zst$
+Regex=\\.(diff|patch)\\.zst$
 Open=%cd %p/patchfs://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view zst
 
 [ls-lR]
-Regex=(^|\.)ls-?lR(\.gz|Z|bz2)$
+Regex=(^|\\.)ls-?lR(\\.gz|Z|bz2)$
 Open=%cd %p/lslR://
 
 [trpm]
@@ -380,7 +380,7 @@
 
 # RPM packages (SuSE uses *.spm for source packages)
 [src.rpm]
-Regex=\.(src\.rpm|spm)$
+Regex=\\.(src\\.rpm|spm)$
 Open=%cd %p/rpm://
 View=%view{ascii} /usr/lib/mc/ext.d/package.sh view src.rpm
 
@@ -390,7 +390,7 @@
 View=%view{ascii} /usr/lib/mc/ext.d/package.sh view rpm
 
 [deb]
-Regex=\.u?deb$
+Regex=\\.u?deb$
 Open=%cd %p/deb://
 View=%view{ascii} /usr/lib/mc/ext.d/package.sh view deb
 
@@ -411,7 +411,7 @@
 View=%view{ascii} /usr/lib/mc/ext.d/misc.sh view iso9660
 
 [ar]
-Regex=\.s?a$
+Regex=\\.s?a$
 Open=%cd %p/uar://
 #Open=%view{ascii} ar tv %f
 View=%view{ascii} /usr/lib/mc/ext.d/misc.sh view ar
@@ -425,12 +425,12 @@
 ### Sources ###
 
 [C/C++]
-Regex=\.(c|cc|cpp|cxx|c\+\+)$
+Regex=\\.(c|cc|cpp|cxx|c\\+\\+)$
 RegexIgnoreCase=true
 Include=editor
 
 [C/C++ header]
-Regex=\.(h|hh|hpp|hxx|h\+\+)$
+Regex=\\.(h|hh|hpp|hxx|h\\+\\+)$
 RegexIgnoreCase=true
 Include=editor
 
@@ -440,7 +440,7 @@
 Include=editor
 
 [Assembler]
-Regex=\.(s|asm)$
+Regex=\\.(s|asm)$
 RegexIgnoreCase=true
 Include=editor
 
@@ -452,19 +452,19 @@
 
 # .so libraries
 [so]
-Regex=\.(so|so\.[0-9\.]*)$
+Regex=\\.(so|so\\.[0-9\\.]*)$
 View=%view{ascii} /usr/lib/mc/ext.d/misc.sh view so
 
 # .dylib libraries
 [dylib]
-Regex=\.(dylib|dylib\.[0-9\.]*)$
+Regex=\\.(dylib|dylib\\.[0-9\\.]*)$
 View=%view{ascii} /usr/lib/mc/ext.d/misc.sh view dylib
 
 
 ### Documentation ###
 
 #[Texinfo]
-#Regex=\.(te?xi|texinfo)$
+#Regex=\\.(te?xi|texinfo)$
 
 [info-by-shell]
 Shell=.info
@@ -497,27 +497,27 @@
 # Manual page
 
 [man.lz]
-Regex=([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|[ln])\.lz$
+Regex=([^0-9]|^[^\\.]*)\\.([1-9][A-Za-z]*|[ln])\\.lz$
 Open=/usr/lib/mc/ext.d/text.sh open man.lz %var{PAGER:more}
 View=%view{ascii,nroff} /usr/lib/mc/ext.d/text.sh view man.lz %var{PAGER:more}
 
 [man.lz4]
-Regex=([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|[ln])\.lz4$
+Regex=([^0-9]|^[^\\.]*)\\.([1-9][A-Za-z]*|[ln])\\.lz4$
 Open=/usr/lib/mc/ext.d/text.sh open man.lz4 %var{PAGER:more}
 View=%view{ascii,nroff} /usr/lib/mc/ext.d/text.sh view man.lz4 %var{PAGER:more}
 
 [man.lzma]
-Regex=([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|[ln])\.lzma$
+Regex=([^0-9]|^[^\\.]*)\\.([1-9][A-Za-z]*|[ln])\\.lzma$
 Open=/usr/lib/mc/ext.d/text.sh open man.lzma %var{PAGER:more}
 View=%view{ascii,nroff} /usr/lib/mc/ext.d/text.sh view man.lzma %var{PAGER:more}
 
 [man.xz]
-Regex=([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|[ln])\.xz$
+Regex=([^0-9]|^[^\\.]*)\\.([1-9][A-Za-z]*|[ln])\\.xz$
 Open=/usr/lib/mc/ext.d/text.sh open man.xz %var{PAGER:more}
 View=%view{ascii,nroff} /usr/lib/mc/ext.d/text.sh view man.xz %var{PAGER:more}
 
 [man.zst]
-Regex=([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|[ln])\.zst$
+Regex=([^0-9]|^[^\\.]*)\\.([1-9][A-Za-z]*|[ln])\\.zst$
 Open=/usr/lib/mc/ext.d/text.sh open man.zst %var{PAGER:more}
 View=%view{ascii,nroff} /usr/lib/mc/ext.d/text.sh view man.zst %var{PAGER:more}
 
@@ -560,13 +560,13 @@
 ### Sound files ###
 
 [sound]
-Regex=\.(wav|snd|voc|au|smp|aiff|snd|m4a|ape|aac|wv|spx|flac)$
+Regex=\\.(wav|snd|voc|au|smp|aiff|snd|m4a|ape|aac|wv|spx|flac)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/sound.sh open common
 View=%view{ascii} /usr/lib/mc/ext.d/sound.sh view common
 
 [mod]
-Regex=\.(mod|s3m|xm|it|mtm|669|stm|ult|far)$
+Regex=\\.(mod|s3m|xm|it|mtm|669|stm|ult|far)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/sound.sh open mod
 
@@ -582,7 +582,7 @@
 View=%view{ascii} /usr/lib/mc/ext.d/sound.sh view mp3
 
 [ogg]
-Regex=\.og[gax]$
+Regex=\\.og[gax]$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/sound.sh open ogg
 View=%view{ascii} /usr/lib/mc/ext.d/sound.sh view ogg
@@ -594,7 +594,7 @@
 View=%view{ascii} /usr/lib/mc/ext.d/sound.sh view opus
 
 [midi]
-Regex=\.(midi?|rmid?)$
+Regex=\\.(midi?|rmid?)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/sound.sh open midi
 
@@ -606,7 +606,7 @@
 
 # Play list
 [playlist]
-Regex=\.(m3u|pls)$
+Regex=\\.(m3u|pls)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/sound.sh open playlist
 
@@ -619,7 +619,7 @@
 Include=video
 
 [asf]
-Regex=\.as[fx]$
+Regex=\\.as[fx]$
 RegexIgnoreCase=true
 Include=video
 
@@ -634,12 +634,12 @@
 Include=video
 
 [mov]
-Regex=\.(mov|qt)$
+Regex=\\.(mov|qt)$
 RegexIgnoreCase=true
 Include=video
 
 [mp4]
-Regex=\.(mp4|m4v|mpe?g)$
+Regex=\\.(mp4|m4v|mpe?g)$
 RegexIgnoreCase=true
 Include=video
 
@@ -665,7 +665,7 @@
 Include=video
 
 [fli]
-Regex=\.fl[icv]$
+Regex=\\.fl[icv]$
 RegexIgnoreCase=true
 Include=video
 
@@ -675,7 +675,7 @@
 Include=video
 
 [realaudio]
-Regex=\.ra?m$
+Regex=\\.ra?m$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/video.sh open ram
 
@@ -688,7 +688,7 @@
 ### Documents ###
 
 [html]
-Regex=\.html?$
+Regex=\\.html?$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/web.sh open html
 View=%view{ascii} /usr/lib/mc/ext.d/web.sh view html
@@ -700,7 +700,7 @@
 
 # StarOffice 6 and OpenOffice.org formats
 [OpenOffice.org]
-Regex=\.(odt|fodt|ott|sxw|stw|ods|fods|ots|sxc|stc|odp|fodp|otp|sxi|sti|odg|fodg|otg|sxd|std|odb|odf|sxm|odm|sxg)$
+Regex=\\.(odt|fodt|ott|sxw|stw|ods|fods|ots|sxc|stc|odp|fodp|otp|sxi|sti|odg|fodg|otg|sxd|std|odb|odf|sxm|odm|sxg)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open ooffice
 View=%view{ascii} /usr/lib/mc/ext.d/doc.sh view odt
@@ -722,21 +722,21 @@
 
 # Microsoft Word Document
 [msdoc-by-shell]
-Regex=\.(do[ct]|wri|docx)$
+Regex=\\.(do[ct]|wri|docx)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open msdoc
 View=%view{ascii} /usr/lib/mc/ext.d/doc.sh view msdoc
 
 # Microsoft Excel Worksheet
 [msxls-by-shell]
-Regex=\.(xl[sw]|xlsx)$
+Regex=\\.(xl[sw]|xlsx)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open msxls
 View=%view{ascii} /usr/lib/mc/ext.d/doc.sh view msxls
 
 # Microsoft PowerPoint Presentation
 [msppt]
-Regex=\.(pp[ts]|pptx)$
+Regex=\\.(pp[ts]|pptx)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open msppt
 View=%view{ascii} /usr/lib/mc/ext.d/doc.sh view msppt
@@ -753,25 +753,25 @@
 Include=editor
 
 [markdown]
-Regex=\.mk?d$
+Regex=\\.mk?d$
 RegexIgnoreCase=true
 Include=editor
 
 [djvu]
-Regex=\.djvu?$
+Regex=\\.djvu?$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open djvu
 View=%view{ascii} /usr/lib/mc/ext.d/doc.sh view djvu
 
 # Comic Books
 [cbr]
-Regex=\.cb[zr]$
+Regex=\\.cb[zr]$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open comic
 
 # Epup, mobi, fb2
 [ebook]
-Regex=\.(epub|mobi|fb2)$
+Regex=\\.(epub|mobi|fb2)$
 RegexIgnoreCase=true
 Open=/usr/lib/mc/ext.d/doc.sh open ebook
 View=%view{ascii} /usr/lib/mc/ext.d/doc.sh view ebook
@@ -790,7 +790,7 @@
 
 # Makefile.PL (MakeMaker)
 [Makefile.pl]
-Regex=^Makefile\.(PL|pl)$
+Regex=^Makefile\\.(PL|pl)$
 Open=%var{PERL:perl} %f
 
 [Makefile]
@@ -805,7 +805,7 @@
 
 # REXX script
 [rexx]
-Regex=\.(rexx?|cmd)$
+Regex=\\.(rexx?|cmd)$
 Open=rexx %f %{Enter parameters};echo "Press ENTER";read y
 
 # Disk images for Commodore computers (VIC20, C64, C128)
@@ -1036,47 +1036,47 @@
 ### Plain compressed files ###
 
 [gzip]
-Type=\(gzip compressed
+Type=gzip compressed
 Open=/usr/lib/mc/ext.d/archive.sh view gz %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view gz
 
 [bzip]
-Type=\(bzip compressed
+Type=bzip compressed
 Open=/usr/lib/mc/ext.d/archive.sh view bzip %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view bzip
 
 [bzip2]
-Type=\(bzip2 compressed
+Type=bzip2 compressed
 Open=/usr/lib/mc/ext.d/archive.sh view bzip2 %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view bz2
 
 [compress]
-Type=\(compress'd
+Type=compress'd
 Open=/usr/lib/mc/ext.d/archive.sh view gz %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view gz
 
 [lz]
-Type=\(lzip compressed
+Type=lzip compressed
 Open=/usr/lib/mc/ext.d/archive.sh view lz %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view lz
 
 [lzma]
-Type=\(LZMA compressed
+Type=LZMA compressed
 Open=/usr/lib/mc/ext.d/archive.sh view lzma %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view lzma
 
 [xz]
-Type=\(XZ compressed
+Type=XZ compressed
 Open=/usr/lib/mc/ext.d/archive.sh view xz %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view xz
 
 [zstd]
-Type=\(Zstandard compressed
+Type=Zstandard compressed
 Open=/usr/lib/mc/ext.d/archive.sh view zst %var{PAGER:more}
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view zst
 
 [zip-by-type]
-Type=\(Zip archive
+Type=Zip archive
 Open=%cd %p/uzip://
 View=%view{ascii} /usr/lib/mc/ext.d/archive.sh view zip
 
