--- docker-compose-gpu.yml	2024-10-18 11:52:32.000000000 +0200
+++ docker-compose-gpu.yml.obsidian	2024-10-18 11:51:23.743424100 +0200
@@ -7,6 +7,26 @@
 #
 # ----------------------------------------------------------------------------------------
 services:
+
+  # obsidian notebook for managing markdown documentation
+  obsidian:
+    image: lscr.io/linuxserver/obsidian:latest
+    container_name: obsidian
+    security_opt:
+      - seccomp:unconfined #optional
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Etc/UTC
+    volumes:
+      - obsidian:/config
+      - workspace:/config/workspace
+    ports:
+      - 3000:3000
+      - 3001:3001
+    shm_size: "1gb"
+    restart: unless-stopped
+
   # run jupyterlab on a local port 8888
   jupyterlab:
     container_name: lab-gpu
@@ -48,6 +68,7 @@
   workspace:
   certs:
   home:
+  obsidian:
 
 networks:
   frontend:
